projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -26,30 +26,45 @@
   color: inherit;
   border: none;
   border-radius: tokens.$cds-alias-object-border-radius-100;
-
   background: none;
   cursor: pointer;
 }
 
 @mixin generate-calendar-focus-style() {
-  &:hover,
-  &:focus {
-    background: datepicker-variables.$clr-calendar-btn-hover-focus-color;
+  &:hover {
+    background: datepicker-variables.$clr-calendar-btn-hover-color;
   }
-
   &:focus {
-    //Only because we have defined a background color for focus above
+    border: tokens.$cds-global-space-1 solid datepicker-variables.$clr-calendar-btn-focus-border-color !important;
     outline: 0;
```

You can just set the proper value on the outline instead of adding a border for focus state
```suggestion
    outline: datepicker-variables.$clr-calendar-btn-focus-border-color solid tokens.$cds-global-space-1;
```